core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala

```diff
@@ -193,18 +210,6 @@ object SharedSQLContext extends Logging {
       _spark = _sparkSession
     }
 
-  private def initializeJDBC(): Unit =
-    if (_sparkJDBC == null) {
-      _sparkJDBC = _sparkSession
-    }
-
-  protected def initializeTiContext(): Unit =
```

Comment:
We do not need to initialize TiContext for tests anymore because it is already present in SparkExtensions